src/components/paragraph/stanford-lists.tsx

```diff
@@ -140,6 +144,12 @@ const getViewPagedItems = cache(async (viewId: string, displayId: string, contex
       items = graphqlResponse.stanfordSharedTags?.results as unknown as NodeUnion[]
       break
 
+    case 'sul_shared_tag_events--card_grid':
+      filters = getViewFilters(['term_node_taxonomy_name_depth', 'type'], contextualFilter)
+      graphqlResponse = await client.stanfordSharedTags({filters, ...queryVariables});
```

Comment:
wrong query. See the `https://github.com/SU-SWS/sulgryphon-nextjs/blob/1.x/src/lib/gql/view-queries.drupal.gql` to start